src/framework/theme/styles/themes/_mapping.scss

```diff
@@ -134,6 +134,7 @@ $eva-mapping: (
 
   menu-item-border-radius: 0,
   menu-item-padding: 0.75rem 1rem,
+  menu-item-badge-padding: 0.3rem,
```

Comment:
We have a badge dot mode padding variable in the every component (tabs, menu, action). Couldn't we specify this padding in the badge itself?